tests/agents/test_ollama_agent.py

```
@@ -14,134 +17,114 @@
     """Test model information methods."""
     print("\n=== Testing Model Information ===")
-    
+
     try:
         available = OllamaAgent.get_available_models()
         print(f"Available models ({len(available)}):")
         for model in available:
             print(f"  - {model}")
-            
+
         print(f"\nRecommended models:")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)